### PR TITLE
Add mnemonic generator

### LINKED ISSUE #38: Provide a way to generate a mnemonic/address pair for faucet
It would be very nice if we could generate a mnemonic and then an address by running a faucet cli command. Even better if that's packaged in a docker container for users to be able to run it without installing all the dependencies.

Basic idea is `faucet gen_mnemonic` outputs a mnemonic
`MNEMONIC="blah blah" faucet address` outputs a corresponding address

That will simplify a lot our readme and setup  of devnets for 3rd party devs.

### PR DESCRIPTION
**Pull request by webmaster128**
We can now do:

```
$ ./bin/iov-faucet generate bns some-test-chain-id
FAUCET_MNEMONIC="excuse decade frost dentist accident short feed private silent caution private vehicle"

Created token holder  (m/1229936198'/1'/0'/0'): tiov1ysfqjpxk62xtmtp4mqtnevaxdqz46qs7la2zqx
Created distributor 1 (m/1229936198'/1'/0'/1'): tiov1ujjqlp3tzjq374k6v72u6tjduh0cd6u39a4zlx
Created distributor 2 (m/1229936198'/1'/0'/2'): tiov1uyq9x9wm53dsyufst9zy8qjsqc65dnnx3se4sc
Created distributor 3 (m/1229936198'/1'/0'/3'): tiov13xgrrkyu2a6yp55lhfle947v03w6q27v8s9hjx
Created distributor 4 (m/1229936198'/1'/0'/4'): tiov1zlj4gkuvznpha0tggkzvepuw3st27zuk8l8rrs
Created distributor 5 (m/1229936198'/1'/0'/5'): tiov1k2lch3nmk4ps5pzkp09e6j58pk596sf8wpyday
```

or

```
$ ./bin/iov-faucet generate lisk test-chain
FAUCET_MNEMONIC="remember service tortoise uniform crawl human swift meat sick coil assault hire"

Created token holder  (m/1229936198'/1'/0'/0'): 4823460794321468344L
Created distributor 1 (m/1229936198'/1'/0'/1'): 4753686483343681383L
Created distributor 2 (m/1229936198'/1'/0'/2'): 3043522886157615371L
Created distributor 3 (m/1229936198'/1'/0'/3'): 13250471228358064982L
Created distributor 4 (m/1229936198'/1'/0'/4'): 9422411317515608937L
Created distributor 5 (m/1229936198'/1'/0'/5'): 9252840493307402670L
```

Closes #38

### CODE REVIEW

File: src/actions/generate.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export async function generate(args: ReadonlyArray<string>): Promise<void> {
+  if (args.length < 2) {
+    throw Error(`Not enough arguments for action 'generate'. See README for arguments.`);
+  }
+  const codecName = codecFromString(args[0]);
+  const chainId = args[1] as ChainId;
+
+  const mnemonic = Bip39.encode(await Random.getBytes(16)).asString();
+  console.log(`FAUCET_MNEMONIC="${mnemonic}"`);
+  console.log("");
```
Comment (willclarktech) [inline src/actions/generate.ts]: What's this for?

File: src/actions/generate.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { ChainId } from "@iov/bcp";
+import { Bip39, Random } from "@iov/crypto";
+import { UserProfile } from "@iov/keycontrol";
+
+import { codecFromString } from "../codec";
+import { setSecretAndCreateIdentities } from "../profile";
+
+export async function generate(args: ReadonlyArray<string>): Promise<void> {
+  if (args.length < 2) {
+    throw Error(`Not enough arguments for action 'generate'. See README for arguments.`);
```
Comment (willclarktech) [inline src/actions/generate.ts]: `help` is probably more useful for a user at this point.

File: src/actions/generate.ts (hunk 1 above)
Comment (webmaster128) [reply src/actions/generate.ts]: To produce the empty line you see in the example output I posted in this PR's description

File: src/actions/generate.ts (hunk 2 above)
Comment (webmaster128) [reply src/actions/generate.ts]: Yeah, I was thinking about that too. Will update

**Pull request merged**